test/stack_test.c

```diff
@@ -90,82 +92,67 @@ static int test_int_stack(void)
 
     /* Check push and num */
     for (i = 0; i < n; i++) {
-        if (sk_sint_num(s) != i) {
+        if (!int_eq(sk_sint_num(s), i, "int stack size %d", i)) {
             fprintf(stderr, "test int stack size %d\n", i);
```

Comment:
Oops, I thought I'd replaced this already :(  It will go away.  @jon-oracle found it soon after I made the initial commit.